docs/getting_started.md

```diff
@@ -359,5 +359,81 @@ At this point, our hello-world application is fully functional, load balanced, s
 #### Unbind
 
 #### Deprovision
+In the deprovision task, we need to get destroy all resources provisioned from the provision task, usually in reverse order.  The exception is the namespace we created, since users may have added other resources in the project and we may not want to delete resources created by other means.
+
+To add the deprovision action, we'll add a yaml file `deprovision.yml` inside the `playbooks` directory.
+```
+my-apb/
+├── apb.yml
+├── Dockerfile
+├── playbooks
+│   ├── deprovision.yml
+│   └── provision.yml
+└── roles
+    └── provision-my-apb
+        └── tasks
+            └── main.yml
+
+```
+
+Now let's paste in the following code into `playbooks/deprovision.yml`:
+```yaml
+- name: my-apb deprovision
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  roles:
+  - role: ansible.kubernetes-modules
+    install_python_requirements: no
+  - role: deprovision-my-apb
+    playbook_debug: false
+```
+The contents looks the same as the provision task, except it's calling a different role which does the actual work.  Let's create that role now. Create a new file called `main.yml` inside a `roles/deprovision-my-apb` directory.
+```
+mkdir -p roles/deprovision-my-apb/tasks
+touch roles/deprovision-my-apb/tasks/main.yml
+```
+Your directory structure should now look like:
+```
+my-apb/
+├── apb.yml
+├── Dockerfile
+├── playbooks
+│   ├── deprovision.yml
+│   └── provision.yml
+└── roles
+    ├── deprovision-my-apb
+    │   └── tasks
+    │       └── main.yml
+    └── provision-my-apb
+        └── tasks
+            └── main.yml
+```
+Let's paste in the following code in `roles/deprovision-my-apb/tasks/main.yml` and then we'll take a look at it.
+```yaml
+- openshift_v1_route:
+    name: hello-world
+    namespace: '{{ namespace }}'
+    state: absent
+
+- k8s_v1_service:
+    name: hello-world
+    namespace: '{{ namespace }}'
+    state: absent
+
+- openshift_v1_deployment_config:
+    name: hello-world
+    namespace: '{{ namespace }}'
+    state: absent
+```
+In the `provision.yml` created earlier, we created the deployment config, service, then route, so for the **_deprovision_** action, we'll want to delete the resources in reverse.  We do so by identifying the resource by `namespace` and `name` and then marking it as `state: absent`.  That's all there is to it.
```

Comment:
I would even remove 'the'

In `provision.yml`, created earlier, we created the deployment config, service, and then the route. For the deprovision action, we'll want to delete the resources in reverse order.